src/global/project/manifest.rs

```diff
@@ -488,12 +488,71 @@ impl Manifest {
         *shortcuts_array = existing_shortcuts.iter().collect();
 
         tracing::debug!(
-            "Added channel {} for environment {env_name} in toml document",
-            console::style(shortcut.as_normalized()).green()
+            "Added shortcut {} for environment {} in toml document",
+            console::style(shortcut.as_normalized()).green(),
+            env_name.fancy_display()
         );
         Ok(())
     }
 
+    /// Removes shortcut from the manifest of any environment
+    ///
+    /// Returns true if the shortcut was removed, false if it didn't exist
```

Comment:
This function doesn't return a boolean

Reply:
Oops, in one of my versions it used to do that ;)